<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<style>body{background-color:white;}</style>


</head>
<body>
<div id="pffadbppne" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
  <style>#pffadbppne table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#pffadbppne thead, #pffadbppne tbody, #pffadbppne tfoot, #pffadbppne tr, #pffadbppne td, #pffadbppne th {
  border-style: none;
}

#pffadbppne p {
  margin: 0;
  padding: 0;
}

#pffadbppne .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: small;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#pffadbppne .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#pffadbppne .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#pffadbppne .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#pffadbppne .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#pffadbppne .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#pffadbppne .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#pffadbppne .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#pffadbppne .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#pffadbppne .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#pffadbppne .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#pffadbppne .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#pffadbppne .gt_spanner_row {
  border-bottom-style: hidden;
}

#pffadbppne .gt_group_heading {
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  color: #FFFFFF;
  background-color: #708090;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#pffadbppne .gt_empty_group_heading {
  padding: 0.5px;
  color: #FFFFFF;
  background-color: #708090;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#pffadbppne .gt_from_md > :first-child {
  margin-top: 0;
}

#pffadbppne .gt_from_md > :last-child {
  margin-bottom: 0;
}

#pffadbppne .gt_row {
  padding-top: 3px;
  padding-bottom: 3px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #CCCCCC;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#pffadbppne .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#pffadbppne .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#pffadbppne .gt_row_group_first td {
  border-top-width: 2px;
}

#pffadbppne .gt_row_group_first th {
  border-top-width: 2px;
}

#pffadbppne .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 3px;
  padding-bottom: 3px;
  padding-left: 5px;
  padding-right: 5px;
}

#pffadbppne .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#pffadbppne .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#pffadbppne .gt_last_summary_row {
  padding-top: 3px;
  padding-bottom: 3px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#pffadbppne .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#pffadbppne .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#pffadbppne .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#pffadbppne .gt_striped {
  background-color: #F2F2F2;
}

#pffadbppne .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#pffadbppne .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#pffadbppne .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#pffadbppne .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#pffadbppne .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#pffadbppne .gt_left {
  text-align: left;
}

#pffadbppne .gt_center {
  text-align: center;
}

#pffadbppne .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#pffadbppne .gt_font_normal {
  font-weight: normal;
}

#pffadbppne .gt_font_bold {
  font-weight: bold;
}

#pffadbppne .gt_font_italic {
  font-style: italic;
}

#pffadbppne .gt_super {
  font-size: 65%;
}

#pffadbppne .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#pffadbppne .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#pffadbppne .gt_indent_1 {
  text-indent: 5px;
}

#pffadbppne .gt_indent_2 {
  text-indent: 10px;
}

#pffadbppne .gt_indent_3 {
  text-indent: 15px;
}

#pffadbppne .gt_indent_4 {
  text-indent: 20px;
}

#pffadbppne .gt_indent_5 {
  text-indent: 25px;
}

#pffadbppne .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#pffadbppne div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>
  <table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
  <thead>
    <tr class="gt_heading">
      <td colspan="2" class="gt_heading gt_title gt_font_normal" style>Last Gift sequencing database</td>
    </tr>
    <tr class="gt_heading">
      <td colspan="2" class="gt_heading gt_subtitle gt_font_normal gt_bottom_border" style>WGS haplotypes  included</td>
    </tr>
    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="a::stub"></th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="font-size: small; font-weight: bold; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: grey70;" scope="col" id="number-of-haplotypes">number of haplotypes</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG01">LG01</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_1" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG01 stub_1_1 number of haplotypes" class="gt_row gt_right">6.0</td></tr>
    <tr><th id="stub_1_2" scope="row" class="gt_row gt_left gt_stub">Adipose (Abdominal)</th>
<td headers="LG01 stub_1_2 number of haplotypes" class="gt_row gt_right gt_striped">6.0</td></tr>
    <tr><th id="stub_1_3" scope="row" class="gt_row gt_left gt_stub">Colon (right)</th>
<td headers="LG01 stub_1_3 number of haplotypes" class="gt_row gt_right">10.0</td></tr>
    <tr><th id="stub_1_4" scope="row" class="gt_row gt_left gt_stub">PBMC</th>
<td headers="LG01 stub_1_4 number of haplotypes" class="gt_row gt_right gt_striped">6.0</td></tr>
    <tr><th id="summary_stub_LG01_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG01 summary_stub_LG01_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">28</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG03">LG03</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_5" scope="row" class="gt_row gt_left gt_stub">Lymph Node (other)</th>
<td headers="LG03 stub_1_5 number of haplotypes" class="gt_row gt_right">10.0</td></tr>
    <tr><th id="stub_1_6" scope="row" class="gt_row gt_left gt_stub">PBMC</th>
<td headers="LG03 stub_1_6 number of haplotypes" class="gt_row gt_right gt_striped">8.0</td></tr>
    <tr><th id="summary_stub_LG03_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG03 summary_stub_LG03_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">18</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG04">LG04</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_7" scope="row" class="gt_row gt_left gt_stub">Duodenum</th>
<td headers="LG04 stub_1_7 number of haplotypes" class="gt_row gt_right">11.0</td></tr>
    <tr><th id="stub_1_8" scope="row" class="gt_row gt_left gt_stub">Esophagus</th>
<td headers="LG04 stub_1_8 number of haplotypes" class="gt_row gt_right gt_striped">5.0</td></tr>
    <tr><th id="summary_stub_LG04_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG04 summary_stub_LG04_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">16</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG05">LG05</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_9" scope="row" class="gt_row gt_left gt_stub">Colon (right)</th>
<td headers="LG05 stub_1_9 number of haplotypes" class="gt_row gt_right">1.0</td></tr>
    <tr><th id="stub_1_10" scope="row" class="gt_row gt_left gt_stub">Liver</th>
<td headers="LG05 stub_1_10 number of haplotypes" class="gt_row gt_right gt_striped">2.0</td></tr>
    <tr><th id="summary_stub_LG05_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG05 summary_stub_LG05_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">3</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG06">LG06</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_11" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG06 stub_1_11 number of haplotypes" class="gt_row gt_right">1.0</td></tr>
    <tr><th id="stub_1_12" scope="row" class="gt_row gt_left gt_stub">Testes</th>
<td headers="LG06 stub_1_12 number of haplotypes" class="gt_row gt_right gt_striped">1.0</td></tr>
    <tr><th id="summary_stub_LG06_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG06 summary_stub_LG06_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">2</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG08">LG08</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_13" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG08 stub_1_13 number of haplotypes" class="gt_row gt_right">24.0</td></tr>
    <tr><th id="stub_1_14" scope="row" class="gt_row gt_left gt_stub">Lymph Node (para-aortic) - tumor</th>
<td headers="LG08 stub_1_14 number of haplotypes" class="gt_row gt_right gt_striped">6.0</td></tr>
    <tr><th id="summary_stub_LG08_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG08 summary_stub_LG08_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">30</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG12">LG12</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_15" scope="row" class="gt_row gt_left gt_stub">Rectum</th>
<td headers="LG12 stub_1_15 number of haplotypes" class="gt_row gt_right">2.0</td></tr>
    <tr><th id="summary_stub_LG12_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG12 summary_stub_LG12_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">2</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG14">LG14</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_16" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG14 stub_1_16 number of haplotypes" class="gt_row gt_right gt_striped">3.0</td></tr>
    <tr><th id="stub_1_17" scope="row" class="gt_row gt_left gt_stub">Jejunum</th>
<td headers="LG14 stub_1_17 number of haplotypes" class="gt_row gt_right">8.0</td></tr>
    <tr><th id="stub_1_18" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG14 stub_1_18 number of haplotypes" class="gt_row gt_right gt_striped">8.0</td></tr>
    <tr><th id="stub_1_19" scope="row" class="gt_row gt_left gt_stub">Occipital Cortex</th>
<td headers="LG14 stub_1_19 number of haplotypes" class="gt_row gt_right">4.0</td></tr>
    <tr><th id="summary_stub_LG14_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG14 summary_stub_LG14_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">23</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG15">LG15</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_20" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG15 stub_1_20 number of haplotypes" class="gt_row gt_right gt_striped">13.0</td></tr>
    <tr><th id="summary_stub_LG15_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG15 summary_stub_LG15_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">13</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG16">LG16</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_21" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG16 stub_1_21 number of haplotypes" class="gt_row gt_right">7.0</td></tr>
    <tr><th id="stub_1_22" scope="row" class="gt_row gt_left gt_stub">Jejunum</th>
<td headers="LG16 stub_1_22 number of haplotypes" class="gt_row gt_right gt_striped">36.0</td></tr>
    <tr><th id="stub_1_23" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG16 stub_1_23 number of haplotypes" class="gt_row gt_right">39.0</td></tr>
    <tr><th id="summary_stub_LG16_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG16 summary_stub_LG16_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">82</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG18">LG18</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_24" scope="row" class="gt_row gt_left gt_stub">Colon (right)</th>
<td headers="LG18 stub_1_24 number of haplotypes" class="gt_row gt_right gt_striped">2.0</td></tr>
    <tr><th id="stub_1_25" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG18 stub_1_25 number of haplotypes" class="gt_row gt_right">1.0</td></tr>
    <tr><th id="stub_1_26" scope="row" class="gt_row gt_left gt_stub">Duodenum</th>
<td headers="LG18 stub_1_26 number of haplotypes" class="gt_row gt_right gt_striped">7.0</td></tr>
    <tr><th id="stub_1_27" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG18 stub_1_27 number of haplotypes" class="gt_row gt_right">3.0</td></tr>
    <tr><th id="summary_stub_LG18_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG18 summary_stub_LG18_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">13</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG21">LG21</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_28" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG21 stub_1_28 number of haplotypes" class="gt_row gt_right gt_striped">2.0</td></tr>
    <tr><th id="stub_1_29" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG21 stub_1_29 number of haplotypes" class="gt_row gt_right">8.0</td></tr>
    <tr><th id="stub_1_30" scope="row" class="gt_row gt_left gt_stub">Duodenum</th>
<td headers="LG21 stub_1_30 number of haplotypes" class="gt_row gt_right gt_striped">9.0</td></tr>
    <tr><th id="summary_stub_LG21_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG21 summary_stub_LG21_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">19</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG22">LG22</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_31" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG22 stub_1_31 number of haplotypes" class="gt_row gt_right">5.0</td></tr>
    <tr><th id="stub_1_32" scope="row" class="gt_row gt_left gt_stub">Duodenum</th>
<td headers="LG22 stub_1_32 number of haplotypes" class="gt_row gt_right gt_striped">15.0</td></tr>
    <tr><th id="stub_1_33" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG22 stub_1_33 number of haplotypes" class="gt_row gt_right">2.0</td></tr>
    <tr><th id="stub_1_34" scope="row" class="gt_row gt_left gt_stub">Lymph Node (paratracheal)</th>
<td headers="LG22 stub_1_34 number of haplotypes" class="gt_row gt_right gt_striped">8.0</td></tr>
    <tr><th id="summary_stub_LG22_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG22 summary_stub_LG22_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">30</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG23">LG23</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_35" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG23 stub_1_35 number of haplotypes" class="gt_row gt_right">3.0</td></tr>
    <tr><th id="stub_1_36" scope="row" class="gt_row gt_left gt_stub">Jejunum</th>
<td headers="LG23 stub_1_36 number of haplotypes" class="gt_row gt_right gt_striped">11.0</td></tr>
    <tr><th id="stub_1_37" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG23 stub_1_37 number of haplotypes" class="gt_row gt_right">8.0</td></tr>
    <tr><th id="summary_stub_LG23_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG23 summary_stub_LG23_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">22</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG24">LG24</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_38" scope="row" class="gt_row gt_left gt_stub">Colon (right)</th>
<td headers="LG24 stub_1_38 number of haplotypes" class="gt_row gt_right gt_striped">14.0</td></tr>
    <tr><th id="stub_1_39" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG24 stub_1_39 number of haplotypes" class="gt_row gt_right">5.0</td></tr>
    <tr><th id="stub_1_40" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG24 stub_1_40 number of haplotypes" class="gt_row gt_right gt_striped">3.0</td></tr>
    <tr><th id="summary_stub_LG24_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG24 summary_stub_LG24_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">22</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG25">LG25</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_41" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG25 stub_1_41 number of haplotypes" class="gt_row gt_right">9.0</td></tr>
    <tr><th id="summary_stub_LG25_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG25 summary_stub_LG25_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">9</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG26">LG26</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_42" scope="row" class="gt_row gt_left gt_stub">Colon (right)</th>
<td headers="LG26 stub_1_42 number of haplotypes" class="gt_row gt_right gt_striped">7.0</td></tr>
    <tr><th id="stub_1_43" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG26 stub_1_43 number of haplotypes" class="gt_row gt_right">1.0</td></tr>
    <tr><th id="stub_1_44" scope="row" class="gt_row gt_left gt_stub">Duodenum</th>
<td headers="LG26 stub_1_44 number of haplotypes" class="gt_row gt_right gt_striped">4.0</td></tr>
    <tr><th id="stub_1_45" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG26 stub_1_45 number of haplotypes" class="gt_row gt_right">8.0</td></tr>
    <tr><th id="stub_1_46" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG26 stub_1_46 number of haplotypes" class="gt_row gt_right gt_striped">7.0</td></tr>
    <tr><th id="summary_stub_LG26_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG26 summary_stub_LG26_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">27</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG28">LG28</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_47" scope="row" class="gt_row gt_left gt_stub">Duodenum</th>
<td headers="LG28 stub_1_47 number of haplotypes" class="gt_row gt_right">5.0</td></tr>
    <tr><th id="stub_1_48" scope="row" class="gt_row gt_left gt_stub">Lymph Node (paratracheal)</th>
<td headers="LG28 stub_1_48 number of haplotypes" class="gt_row gt_right gt_striped">5.0</td></tr>
    <tr><th id="summary_stub_LG28_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG28 summary_stub_LG28_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">10</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG29">LG29</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_49" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG29 stub_1_49 number of haplotypes" class="gt_row gt_right">12.0</td></tr>
    <tr><th id="stub_1_50" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG29 stub_1_50 number of haplotypes" class="gt_row gt_right gt_striped">5.0</td></tr>
    <tr><th id="stub_1_51" scope="row" class="gt_row gt_left gt_stub">Jejunum</th>
<td headers="LG29 stub_1_51 number of haplotypes" class="gt_row gt_right">8.0</td></tr>
    <tr><th id="summary_stub_LG29_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG29 summary_stub_LG29_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">25</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG30">LG30</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_52" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG30 stub_1_52 number of haplotypes" class="gt_row gt_right gt_striped">7.0</td></tr>
    <tr><th id="stub_1_53" scope="row" class="gt_row gt_left gt_stub">Duodenum</th>
<td headers="LG30 stub_1_53 number of haplotypes" class="gt_row gt_right">7.0</td></tr>
    <tr><th id="stub_1_54" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG30 stub_1_54 number of haplotypes" class="gt_row gt_right gt_striped">11.0</td></tr>
    <tr><th id="summary_stub_LG30_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG30 summary_stub_LG30_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">25</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG31">LG31</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_55" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG31 stub_1_55 number of haplotypes" class="gt_row gt_right">1.0</td></tr>
    <tr><th id="stub_1_56" scope="row" class="gt_row gt_left gt_stub">Duodenum</th>
<td headers="LG31 stub_1_56 number of haplotypes" class="gt_row gt_right gt_striped">7.0</td></tr>
    <tr><th id="stub_1_57" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG31 stub_1_57 number of haplotypes" class="gt_row gt_right">6.0</td></tr>
    <tr><th id="summary_stub_LG31_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG31 summary_stub_LG31_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">14</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG33">LG33</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_58" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG33 stub_1_58 number of haplotypes" class="gt_row gt_right gt_striped">8.0</td></tr>
    <tr><th id="stub_1_59" scope="row" class="gt_row gt_left gt_stub">Duodenum</th>
<td headers="LG33 stub_1_59 number of haplotypes" class="gt_row gt_right">6.0</td></tr>
    <tr><th id="stub_1_60" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG33 stub_1_60 number of haplotypes" class="gt_row gt_right gt_striped">2.0</td></tr>
    <tr><th id="summary_stub_LG33_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG33 summary_stub_LG33_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">16</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG34">LG34</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_61" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG34 stub_1_61 number of haplotypes" class="gt_row gt_right">8.0</td></tr>
    <tr><th id="stub_1_62" scope="row" class="gt_row gt_left gt_stub">Duodenum</th>
<td headers="LG34 stub_1_62 number of haplotypes" class="gt_row gt_right gt_striped">3.0</td></tr>
    <tr><th id="stub_1_63" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG34 stub_1_63 number of haplotypes" class="gt_row gt_right">4.0</td></tr>
    <tr><th id="summary_stub_LG34_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG34 summary_stub_LG34_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">15</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG37">LG37</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_64" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG37 stub_1_64 number of haplotypes" class="gt_row gt_right gt_striped">2.0</td></tr>
    <tr><th id="stub_1_65" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG37 stub_1_65 number of haplotypes" class="gt_row gt_right">6.0</td></tr>
    <tr><th id="summary_stub_LG37_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG37 summary_stub_LG37_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">8</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG38">LG38</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_66" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG38 stub_1_66 number of haplotypes" class="gt_row gt_right gt_striped">26.0</td></tr>
    <tr><th id="stub_1_67" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG38 stub_1_67 number of haplotypes" class="gt_row gt_right">34.0</td></tr>
    <tr><th id="summary_stub_LG38_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG38 summary_stub_LG38_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">60</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG39">LG39</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_68" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG39 stub_1_68 number of haplotypes" class="gt_row gt_right gt_striped">24.0</td></tr>
    <tr><th id="stub_1_69" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG39 stub_1_69 number of haplotypes" class="gt_row gt_right">16.0</td></tr>
    <tr><th id="summary_stub_LG39_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG39 summary_stub_LG39_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">40</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG40">LG40</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_70" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG40 stub_1_70 number of haplotypes" class="gt_row gt_right gt_striped">7.0</td></tr>
    <tr><th id="stub_1_71" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG40 stub_1_71 number of haplotypes" class="gt_row gt_right">20.0</td></tr>
    <tr><th id="summary_stub_LG40_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG40 summary_stub_LG40_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">27</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG42">LG42</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_72" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG42 stub_1_72 number of haplotypes" class="gt_row gt_right gt_striped">6.0</td></tr>
    <tr><th id="summary_stub_LG42_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG42 summary_stub_LG42_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">6</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG43">LG43</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_73" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG43 stub_1_73 number of haplotypes" class="gt_row gt_right">24.0</td></tr>
    <tr><th id="stub_1_74" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG43 stub_1_74 number of haplotypes" class="gt_row gt_right gt_striped">2.0</td></tr>
    <tr><th id="summary_stub_LG43_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG43 summary_stub_LG43_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">26</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG44">LG44</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_75" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG44 stub_1_75 number of haplotypes" class="gt_row gt_right">18.0</td></tr>
    <tr><th id="stub_1_76" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG44 stub_1_76 number of haplotypes" class="gt_row gt_right gt_striped">1.0</td></tr>
    <tr><th id="stub_1_77" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG44 stub_1_77 number of haplotypes" class="gt_row gt_right">7.0</td></tr>
    <tr><th id="summary_stub_LG44_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG44 summary_stub_LG44_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">26</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG48">LG48</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_78" scope="row" class="gt_row gt_left gt_stub">Jejunum</th>
<td headers="LG48 stub_1_78 number of haplotypes" class="gt_row gt_right gt_striped">7.0</td></tr>
    <tr><th id="stub_1_79" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG48 stub_1_79 number of haplotypes" class="gt_row gt_right">34.0</td></tr>
    <tr><th id="stub_1_80" scope="row" class="gt_row gt_left gt_stub">Basal Ganglia</th>
<td headers="LG48 stub_1_80 number of haplotypes" class="gt_row gt_right gt_striped">38.0</td></tr>
    <tr><th id="summary_stub_LG48_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG48 summary_stub_LG48_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">79</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG49">LG49</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_81" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG49 stub_1_81 number of haplotypes" class="gt_row gt_right">7.0</td></tr>
    <tr><th id="stub_1_82" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG49 stub_1_82 number of haplotypes" class="gt_row gt_right gt_striped">48.0</td></tr>
    <tr><th id="summary_stub_LG49_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG49 summary_stub_LG49_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">55</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG50">LG50</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_83" scope="row" class="gt_row gt_left gt_stub">Ileum</th>
<td headers="LG50 stub_1_83 number of haplotypes" class="gt_row gt_right">1.0</td></tr>
    <tr><th id="stub_1_84" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG50 stub_1_84 number of haplotypes" class="gt_row gt_right gt_striped">22.0</td></tr>
    <tr><th id="summary_stub_LG50_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG50 summary_stub_LG50_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">23</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG51">LG51</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_85" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG51 stub_1_85 number of haplotypes" class="gt_row gt_right">12.0</td></tr>
    <tr><th id="stub_1_86" scope="row" class="gt_row gt_left gt_stub">Jejunum</th>
<td headers="LG51 stub_1_86 number of haplotypes" class="gt_row gt_right gt_striped">9.0</td></tr>
    <tr><th id="summary_stub_LG51_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG51 summary_stub_LG51_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">21</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG52">LG52</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_87" scope="row" class="gt_row gt_left gt_stub">Jejunum</th>
<td headers="LG52 stub_1_87 number of haplotypes" class="gt_row gt_right">1.0</td></tr>
    <tr><th id="stub_1_88" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG52 stub_1_88 number of haplotypes" class="gt_row gt_right gt_striped">15.0</td></tr>
    <tr><th id="summary_stub_LG52_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG52 summary_stub_LG52_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">16</td></tr>
    <tr class="gt_group_heading_row">
      <th colspan="2" class="gt_group_heading" scope="colgroup" id="LG58">LG58</th>
    </tr>
    <tr class="gt_row_group_first"><th id="stub_1_89" scope="row" class="gt_row gt_left gt_stub">Spleen</th>
<td headers="LG58 stub_1_89 number of haplotypes" class="gt_row gt_right">16.0</td></tr>
    <tr><th id="summary_stub_LG58_1" scope="row" class="gt_row gt_left gt_stub gt_summary_row gt_first_summary_row thick gt_last_summary_row">Total</th>
<td headers="LG58 summary_stub_LG58_1 number of haplotypes" class="gt_row gt_right gt_summary_row gt_first_summary_row thick gt_last_summary_row">16</td></tr>
    <tr><th id="grand_summary_stub_1" scope="row" class="gt_row gt_left gt_stub gt_grand_summary_row gt_first_grand_summary_row gt_last_summary_row">Total</th>
<td headers="grand_summary_stub_1 number of haplotypes" class="gt_row gt_right gt_grand_summary_row gt_first_grand_summary_row gt_last_summary_row">867</td></tr>
  </tbody>
  
</table>
</div>
</body>
</html>
